### PR TITLE
Fix/working version

### PR DESCRIPTION
**Pull request by ManuelTrambert**

### CODE REVIEW

File: __tests__/api.spec.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { API } from '../api';
+import "babel-polyfill";
```
Comment (iGitScor) [inline __tests__/api.spec.js]: That's ok for this iteration but if several tests need `babel-polyfill`, we could use common jest setup

File: package.json [hunk 2]
```diff
@@ -11,6 +11,11 @@
     "@babel/core": "^7.4.5",
     "@babel/node": "^7.4.5",
     "@babel/preset-env": "^7.4.5",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (iGitScor) [inline package.json]: Does this package is deprecated and replaced by `babel-preset-env` ?

Comment (ManuelTrambert) [reply package.json]: Tbh I don't know, it might be useless it was to make unit test works

Comment (iGitScor) [reply package.json]: Ok for me ✔️

**Pull request merged**